tests/simulator/test_vtk_diagnostics.py

```
@@ -191,6 +165,67 @@ def test_dump_diags(self, ndim, interp, simInput):
 
+    @data(*permute({}))
+    @unpack
+    def test_compare_l0_primal_to_phareh5(self, ndim, interp, simInput):
+        print("test_compare_l0_primal_to_phareh5 dim/interp:{}/{}".format(ndim, interp))
+
+        b0 = [[10 for i in range(ndim)], [19 for i in range(ndim)]]
+        simInput["refinement_boxes"] = {"L0": {"B0": b0}}
+        vtk_diags = self._run(ndim, interp, simInput, "test_vtk")
+
+        simInput["diag_options"]["format"] = "phareh5"
+        simulation.diag_options["options"]["dir"] += "/phareh5"
```

Ruff (0.15.17)

[error] Undefined name `simulation`

(F821)